.circleci/config.yml

```diff
@@ -314,7 +316,7 @@ workflows:
           environment: ccq-uat
           context: laa-check-client-qualifies-uat
           requires:
-            - run_specs
+            - coverage
```

Comment:
comment - I think it make sense that `coverage` is now the dependancy that needs to happen before we deploy.